### ISSUE TITLE
CTD on Magiclysm after Attunement to Biotek and casting Synaptic Stimulation

### ISSUE THREAD

**Issue by Profit0004** [CleverRaven/Cataclysm-DDA#66979]
### Describe the bug

CTD on Magiclysm after Attunement to Biotek and casting Synaptic Stimulation.

When cast it causes the game to show a window that a SEGFAULT has occurred. 

Worked fine seconds before reaching attunement altar, only after attuning to Biotek does it crash. 

### Attach save file

[Calavo Gardens-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/12074279/Calavo.Gardens-trimmed.tar.gz)

### Steps to reproduce

1. Have Biotek attunement on character
2. Open spell menu and cast Synaptic Stimulation. (Several other spells cast successfully so I believe it is specific to this spell)
3. Crash. 

### Expected behavior

Not crashing to desktop?

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22621.1992 (22H2)
- Game Version: 5382348 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    No Rail Stations [No_Rail_Stations],
    sees-player, run, crouch, bleed icons, retrodays [sees_player_retro],
    Bionic Professions [package_bionic_professions],
    Blaze Industries [blazeindustries],
    Magiclysm [magiclysm],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills]
]

### Additional context

[crash.log](https://github.com/CleverRaven/Cataclysm-DDA/files/12074283/crash.log)
[debug.log](https://github.com/CleverRaven/Cataclysm-DDA/files/12074284/debug.log)

**Comment by RenechCDDA** [comment]
Duplicate of #66100
